src/main.py

```diff
@@ -88,20 +100,22 @@
             slack_message_text = "*`{0}/{0}` tests passed :white_check_mark:*".format(num_test_group_tests)
         test_group.slack_message.text = slack_message_text
         # Always print output for debugging purposes
-        print(test_group.slack_message.text)
-
-f = open("./test_config.json", "w")
-f.write(str(test_config))
-f.close()
-
+        # print(test_group.slack_message.text)
+if locally_run:
+    f = open("./src/test_config.json", "w")
+    f.write(str(test_config))
+    f.close()
+print("TEST_CONFIG="+str(test_config))
 
 # Send output to server if necessary
 if local_only:
     exit()
 
-CURL_PREFIX = """curl -X POST -H 'Content-type: application/json' --data """
+CURL_PREFIX = """curl -X POST -H 'Content-type: application/json' --silent --output /dev/null --data """
+
 for test_group in test_groups:
-    if copy_config.get(test_group.name)=="ON" or test_config.get(test_group.name)=="ON": # Only output enabled test groups, which include newly failing ones
+    if original_config.get(test_group.name)=="ON" and test_config.get(test_group.name)=="FAILED" or original_config.get(test_group.name)=="FAILED" and test_config.get(test_group.name)=="ON":
```

Comment:
i think we need parentheses here right?
`if a == "ON" and b == "FAILED" or a == "FAILED" and b == "ON"` would only evaluate to True if the last two conditions are true